### DISCUSSION TITLE
Allow spin to be installed via Homebrew

### DISCUSSION THREAD

**Discussion by jaydrogers** [serversideup/spin#29] [Ideas]
# 👉 Describe the problem
[comment]: <> (What are you trying to solve?)
- Some Mac users may want to use `spin` to simply install and manage spin

# 🏆 How to solve this problem
[comment]: <> (Describe the feature that you are proposing and how it will solve the problem)

- [ ] Submit `spin` to be installed via Homebrew
- [ ] See if `spin` is even available and doesn't collide with other projects

# 💯 How do we validate the problem is solved?
[comment]: <> (Explain use cases on how we can measure the success of this implementation)

- [ ] Users will be able to do `brew install spin` to easily install Spin to their machine

**Comment by wblondel** [comment]
FYI the [formula spin](https://formulae.brew.sh/formula/spin) already exists, but it is for another piece of software

**Reply by jaydrogers** [reply]
I saw that. There's a good chance we just keep it with the installer from our website for now. Thanks for the heads up!
